packages/core/src/utils.ts

```diff
@@ -83,6 +83,11 @@ export function shortenEns(ens: string): string {
   return ens.length > 11 ? `${ens.slice(0, 4)}...${ens.slice(-6)}` : ens
 }
 
+export function shortenHexToString(hex: string) : string {
+  const hexToString = (parseInt(hex, 16) / 1e18).toString()
+  return hexToString.length > 5 ?  hexToString.slice(0,6) : hexToString
+}
+
```

Comment:
I think that this can be removed as it is not needed. Also not sure what it is exactly doing? Seems to be doing more than just shortening the string with that divide in there.